osidb/mixins.py

```diff
@@ -831,6 +866,10 @@ def validate(self, raise_validation_error=True, dry_run=False):
                         **(e.params or {}),
                     )
 
+        # standard validations
+        # exclude meta attributes
+        self.full_clean(exclude=["meta_attr"])
```

Comment:
It seems to me we are validating `ArrayField` twice (the first `self.full_clean` validates `ArrayField` only).

Reply:
That is true, I can exclude it here

Reply:
Excluded